src/main/core/Resources/config/services/subscriber.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
```

Comment:
We don't use autowiring/autoconfiguration on this project, to avoid compile-time perf issues due to the big amount of services and circular dependencies we have. 
Given that `public: false` is the default in Symfony 4.4, the `_defaults` block can be removed here, same for other services files introduced in this PR and future ones.

Reply:
Change done :)